### PR TITLE
loader: Fix 2 warnings detected in VS2015.

### PR DESCRIPTION
**Pull request by null77**
warning C4456: declaration of 'i' hides previous local declaration
warning C4244: '=': conversion from 'int' to 'uint16_t', possible loss of data

Please take a look, some very simple fixes.

### CODE REVIEW

File: loader/loader.c [hunk 1]
```diff
@@ -3012,10 +3012,10 @@ void loader_layer_scan(const struct loader_instance *inst,
 
     for (uint32_t manFile = 0; manFile < 2; manFile++) {
         if (NULL != manifest_files[manFile].filename_list) {
-            for (uint32_t i = 0; i < manifest_files[manFile].count; i++) {
-                if (NULL != manifest_files[manFile].filename_list[i]) {
+            for (uint32_t ii = 0; ii < manifest_files[manFile].count; ii++) {
```
Comment (courtney-g) [inline loader/loader.c]: If I'm reading this right, "i" is only used inside the loop, probably should get rid of the uint32_t i at line 2955 and just use a loop local definition of i in the loops.

Comment (MarkY-LunarG) [inline loader/loader.c]: Other than that the change looks good.  Courtney, I'll handle merging this in once he makes that change.  Jon's in the process of teaching me how to integrate the pull requests.

Comment (courtney-g) [inline loader/loader.c]: Sounds good.

Comment (null77) [inline loader/loader.c]: Done, PTAL

**Comment by MarkY-LunarG** [comment]
Request has been merged into master (New commit id: d930517699557e8448e919c68cb8983cfa428201).  Thanks

**Comment by null77** [comment]
Thanks!

**Pull request closed without merging**